sig-k8s-infra/charter.md

```diff
@@ -1,129 +1,184 @@
+otherwise manage all project infrastructure. Given that this is a broad scope,
+we prefer (where possible) to delegate ownership and operation of the 
+code / infrastructure to more directly responsible SIGs or Committees. This
+is largely how the SIG operated during its lifetime as a WG, driving the
+policies and tooling upon which SIG-owned infrastructure operates.
+
+Areas of responsibility include:
+
+- Policy definition and enforcement for areas related to project
+  infrastructure, including:
+  - What is in-scope/out-of-scope for project infrastructure
+  - Who should be allowed access to which parts of project infrastructure,
+    e.g. team definition, vetting criteria, etc.
+  - How infrastructure should be managed, e.g. naming schemes, acceptable
+    tooling or practices, on-call or escalation policies, etc.
+- Configuration management of all resources and service usage within the
+  kubernetes.io GCP Organization, including, but not limited to:
+  - API / Service enablement
+  - BigQuery datasets
+  - DNS records, e.g. for k8s.io, kubernetes.io, and other project-owned domains
+  - GCB usage
+  - GCP projects, instances, images
+  - GCR repositories
+  - GCS buckets
+  - GKE clusters, e.g. community infra cluster, prow build clusters
+  - GSM secrets
+  - Google Groups
+  - IAM roles, service accounts, and policies
+  - KMS keys
+  - Managed Certificates, e.g. for k8s.io, kubernetes.io, and other project-owned
+    domains
+- Reports on infrastructure operation, including:
+  - Anonymized traffic reports to show which parts of our infrastructure
+    are seeing the most use
+  - Auditing reports to show the current configuration of the community's
+    infrastructure
+  - Billing reports to show where the community's infrastructure budget is
+    being spent
+
+In terms of subprojects, this means we own kubernetes/k8s.io and are an
+escalation point of last resort for more tightly scoped subprojects that
+live within this repo.
 
 #### Cross-cutting and Externally Facing Processes
 
-##### Deploying Changes
+We prefer (where possible) to delegate ownership, operation and policy
+definition to SIGs that are more directly responsible for a given area
+of the project. However, we reserve the right to halt infrastructure or
+roll back changes if the project as a whole is being negatively impacted.
+
+Some examples for illustrative purposes
+
+##### Access Policies
+
+- We are responsible for ensuring the appropriate members of a SIG have
+  sufficient permissions to troubleshoot and manage their app or
+  infrastructure.
+- However, we will NOT grant overly broad permissions to an overly broad
+  group of people. We will collaborate with SIGs to ensure access is
+  appropriately scoped.
+- We MAY revoke access in the event of a security-related incident
+
+e.g. SIG Release is responsible for who gets what level of access to
+infrastructure used by the release-engineering subproject to cut a kubernetes
```

Comment:
nit: "Kubernetes"

Reply:
Found this in a few other places, replaced